dalston/functions.php

```diff
@@ -197,33 +197,50 @@ function dalston_editor_styles() {
 function dalston_block_extends() {
 
```

Comment:
@itowhid06, I think it might be better if we remove both the block style scripts _and_ styles from AMP. Without those JS files, the custom block classes (`.is-style-full-bleed` for example) will be missing so the accompanying CSS will be useless anyway. Let’s remove all the `if ( ! dalston_is_amp() ) {…}` conditionals and add something like this at the top of the function:

```
if ( dalston_is_amp() ) {
    return;
}
```

Reply:
Hi @allancole, Thank you very much for your input. I have updated the PR as per your suggestion. Please check.